tests/e2e/run-scanner-v4.bats

```diff
@@ -68,17 +68,167 @@ setup() {
     fi
 
     test_case_no=$(( test_case_no + 1))
+
+    export MAIN_IMAGE_TAG=${MAIN_IMAGE_TAG:-}
```

Comment:
I think image tag and scanner feature flag should be in `setup_file()` like other envs